samples/snippets/create_config.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Game Servers sample for creating a game server config.
+
+Example usage:
+    python create_config.py --project-id <project-id> --deployment-id <deployment-id> --config-id <config-id>
+"""
+
+import argparse
+
+from google.cloud import gaming
+from google.cloud.gaming_v1.types import game_server_configs
+
+FLEET_SPEC = """
```

Comment:
Is there guidance somewhere on how to construct this JSON?

Reply:
I see the protos mention https://agones.dev/site/docs/reference/fleet/, could you add that as a comment here as well? https://github.com/googleapis/python-game-servers/blob/952f47e2094c0246953ae537add751fe0159710b/google/cloud/gaming_v1/types/game_server_configs.py#L199-L202

Reply:
(Apologies for triple posting) I see the link provides a YAML but no JSON equivalent.  If the Agones docs are within in your control it may be nice to provide a JSON version.

Reply:
Per the go canonical example, https://github.com/GoogleCloudPlatform/golang-samples/blob/master/gaming/servers/create_config.go#L28

This fleet spec should be included in the sample range, and have the same comment as the Go fleet has.

Reply:
Added the same comment as the Go sample has. Thanks for point this out.

Reply:
Please move the sample's `# [START cloud_game_servers_config_create]` to above the start of the fleet spec definition.

Reply:
Done. Sorry I missed what you meant in the previous comment.